backend-go/internal/jwt/generator_test.go

```
@@ -0,0 +1,57 @@
+
+		claims, ok := parsedToken.Claims.(*Payload)
+		assert.True(t, ok)
+		assert.Equal(t, dummyUserID, claims.Subject)
+		assert.Equal(t, dummyNickname, claims.Nickname)
+	})
+
+	t.Run("validate refresh token test", func(t *testing.T) {
+		token, err := gen.GenerateRefreshToken(dummyUserID)
+		assert.NoError(t, err)
+
+		parsedToken, err := jwt.ParseWithClaims(token, &Payload{}, func(token *jwt.Token) (interface{}, error) {
```

golangci-lint (1.62.2)

[warning] unused-parameter: parameter 'token' seems to be unused, consider removing or renaming it as _

(revive)